src/formatter/Cli.hx

```diff
@@ -19,6 +19,7 @@ class Cli {
 	var mode:Mode = Format;
 	var exitCode:Int = 0;
 	var lastConfigFileName:Null<String>;
+	var extension:String;
```

Comment:
where does it get initialised to `"hx"`?

Reply:
It gets initialized here:
```haxe
@doc("File extension to use, defaults to hx")
["-e", "--extension"] => function(fileExtension:String) extension = fileExtension?.toString() ?? "hx"
```
If `-e`/`--extension` is not specified, it falls back to `hx`

Reply:
without `-e` the function won't be called, and `?? "hx"` wouldn't apply in that case. not sure you can even pass a `null` `fileExtension` parameter

Reply:
Noted